### DISCUSSION TITLE
Why is the image blurred when oponed within nvim's terminal?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [hpjansson/chafa#159] [Q&A]
I am using [kitty](https://sw.kovidgoyal.net/kitty/) on macOS, and `chara` works perfectly.

However, it becomes blurred if I use [toggleterm.nvim](https://github.com/akinsho/toggleterm.nvim) in Nvim:

<img width="755" alt="image" src="https://github.com/hpjansson/chafa/assets/2310958/db1a64c2-2dc3-4b88-8ce0-86e19502af75">

How to solve this problem?

**Comment by hpjansson** [comment]
Hi, thanks for reporting this. It's happening because the Neovim terminal doesn't support raster graphics. Therefore we emulate it using Unicode characters. If you'd like better graphics support, you should contact the Neovim authors, though note that this would be a lot of work to implement.
